float32 ModelVelX
float32 ModelZ
float32 ModelVelZ
float32 ModelPitch
float32 ModelVelP
float32 ModelDelta
float32 FoilLift
float32 FoilDrag
float32 FoilMoment
float32 HullLift
float32 HullDrag
float32 HullMomentDueToPitch
float32 HullMomentDueToPitchVel
float32 HullHeaveDueToZdot
float32 RestoringMoment

